src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -249,9 +253,13 @@ public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceNa
             }
             catch (HttpRequestException)
             {
-                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider",
-                    $"message:Observer SQL query request failed, query:{query}, statusCode:{response.StatusCode}, response:{result}");
-                throw;
+                loggingResponse = result;
```

Comment:
Throw the exception otherwise it will be looked at as a success

Reply:
Oops, moving too fast